### PR TITLE
Dvm test case

### PR DESCRIPTION
**Pull request by hujw77**
## Add Test Case
- [x] test contract log and anonymous log
- [x] test solidity opcodes
- [x] test net rpc (partial skip)
- [x] test filter rpc (skip)
- [ ] test debug rpc (rpc not implement)

### CODE REVIEW

**Comment by AsceticBear** [comment]
The RPC test seems to run forever ....

File: script_tests/tests/config.js [hunk 1]
```diff
@@ -0,0 +1,523 @@
+const _ = require("underscore");
+
+const config = {
+	rpcMessageId: 1,
+	host: "ws://localhost:9944",
+	// host: 'ws://t1.hkg.itering.com:9944',
```
Comment (AsceticBear) [inline script_tests/tests/config.js:6]: This comment line is useless, please delete it.

File: script_tests/tests/3_test-rpc-filter.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+				address: "0xfd9801e0aa27e54970936aa910a7186fdf5549bc",
+				topics: ["0x01e0aa27e54970936aa910a713", "0x6aa910a7186fdf"],
+			},
+		];
+		currentFilterId = await utils.customRequest("eth_newFilter", params);
+		assert.isNumber(currentFilterId);
+	});
+
+	// it('should return a number as hexstring when all options are passed with address array', async function () {
+	//     const params = [{
```
Comment (AsceticBear) [inline script_tests/tests/3_test-rpc-filter.js:43]: If all of the comment tests need to be skipped, please use `it.skip()` like what you did in other test files. Or delete.

File: script_tests/tests/1_test-transfer.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+const expect = require("chai").expect;
```
Comment (AsceticBear) [inline script_tests/tests/1_test-transfer.js:1]: Why rename these test files name to `1_test-transfer.js `, `2_test-constract-log.js `, `3_test-rpc-filter.js`, `4_test-solidity-opcodes.js `?

File: script_tests/tests/contract/RPCTestContract.sol [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+		return myUint256;
+	}
+
+	function getAddress() public view returns (address ret) {
+		return myAddress;
+	}
+
+	//function getBytes0() returns (bytes0 ret) {
+	//    return myBytes0;
+	//}
```
Comment (AsceticBear) [inline script_tests/tests/contract/RPCTestContract.sol:97]: ?

File: script_tests/tests/1_test-transfer.js (hunk 3 above)
Comment (hujw77) [reply script_tests/tests/1_test-transfer.js:1]: It defines the test order.

File: script_tests/tests/3_test-rpc-filter.js (hunk 2 above)
Comment (hujw77) [reply script_tests/tests/3_test-rpc-filter.js]: done

File: script_tests/tests/contract/RPCTestContract.sol (hunk 4 above)
Comment (hujw77) [reply script_tests/tests/contract/RPCTestContract.sol]: removed

**Comment by hujw77** [comment]
> The RPC test seems to run forever ....

May cause by `it.skip`

**Review by AsceticBear — APPROVED** [review]
LGTM!

**Comment by AurevoirXavier** [comment]
Is this ready for merge?

**Comment by AsceticBear** [comment]
> Is this ready for merge?

LGTM. can be merged.

**Comment by AsceticBear** [comment]
@AurevoirXavier @HackFisher Please check this pr again. The changes looks good to me and i do not have merge authority for this repo.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**